manifests/rule.pp

```diff
@@ -59,8 +59,8 @@
   String $comment = $name,
   Optional[Ferm::Actions] $action = undef,
   Optional[Ferm::Policies] $policy = undef,
-  Optional[Variant[Stdlib::Port,Array[Stdlib::Port]]] $dport = undef,
-  Optional[Variant[Stdlib::Port,Array[Stdlib::Port]]] $sport = undef,
+  Optional[Variant[Stdlib::Port, Array[Stdlib::Port], String[1]]] $dport = undef,
+  Optional[Variant[Stdlib::Port, Array[Stdlib::Port], String[1]]] $sport = undef,
```

Comment:
to not break this in the future, can you add unit tests for the different dport/sport syntax?

Reply:
done